panel/containerPanel.go

```diff
@@ -22,6 +22,7 @@ type ContainerList struct {
 	Data       map[string]interface{}
 	filter     string
 	form       *Form
+	stop       chan int
```

Comment:
Use empty struct channel?
https://dave.cheney.net/2014/03/25/the-empty-struct

Reply:
I'll use empty struct channel!
Thank you!